test/common/quic/platform/quic_platform_test.cc

```diff
@@ -98,12 +98,12 @@ TEST_F(QuicPlatformTest, QuicClientStats) {
   QUIC_CLIENT_HISTOGRAM_ENUM("my.enum.histogram", TestEnum::ONE, TestEnum::COUNT, "doc");
   QUIC_CLIENT_HISTOGRAM_BOOL("my.bool.histogram", false, "doc");
   QUIC_CLIENT_HISTOGRAM_TIMES("my.timing.histogram", QuicTime::Delta::FromSeconds(5),
-                              QuicTime::Delta::FromSeconds(1), QuicTime::Delta::FromSecond(3600),
```

Comment:
Wow, did this code just not compile before, but was hidden inside of the macro magic. Amazing! Thanks for fixing.